.github/workflows/prep-deps.yml

```
@@ -0,0 +1,99 @@
+name: 'Prep Dependencies'
+on:
+  workflow_call:
+    outputs:
+      TEST_CHUNKS:
+        value: ${{ jobs.prepare.outputs.test_chunks }}
+      E2E_CHUNKS:
+        value: ${{ jobs.prepare.outputs.e2e_chunks }}
+      E2E_BUILD_ID:
+        value: ${{ jobs.prepare.outputs.build_iD }}
```

actionlint (1.7.4)

property "build_id" is not defined in object type {build_chunks: string; e2e_build_id: string; e2e_chunks: string; nx_base: string; test_chunks: string}

(expression)